go/docker/build.sh

```diff
@@ -0,0 +1,17 @@
+cat > ./Dockerfile << EOF
```

Comment:
Yep, but you should `wget golang.tar.gz` if you use ubuntu as the base image. and we also need VPN so that we can download the tar file from `storage.googleapis.com`.